client/app/services/user.js

```diff
@@ -16,18 +16,67 @@ function transform(data) {
   return data;
 }
 
-function User($resource, $http) {
+function enableUser(user, toastr, $sanitize) {
+  const userName = $sanitize(user.name);
+  return user.$enable()
+    .then((data) => {
+      toastr.success(
+        `User <b>${userName}</b> is now enabled.`,
+        { allowHtml: true },
+      );
+      return data;
+    })
+    .catch((response) => {
+      let message = response instanceof Error ? response.message : response.statusText;
+      if (!isString(message)) {
+        message = 'Unknown error';
+      }
+      toastr.error(
+        `Cannot enable user <b>${userName}</b><br>${message}`,
+        { allowHtml: true },
```

Comment:
We have user generated content here (`userName`) and allowHtml... does it go through sanitization (by `toastr` maybe)?

Reply:
yes, `toastr` uses `$sanitize` somewhere inside (I checked its sources)